src/System.Text.Json/tests/Serialization/OptionsTests.cs

```diff
@@ -235,5 +240,86 @@ public static void MaxDepthRead()
 
             Assert.Throws<JsonException>(() => JsonSerializer.Deserialize<BasicCompany>(BasicCompany.s_data, options));
         }
+
+        private class TestClassForEncoding
+        {
+            public string MyString { get; set; }
+        }
+
+        // This is a copy of the test data in System.Text.Json.Tests.JsonEncodedTextTests.JsonEncodedTextStringsCustom
+        public static IEnumerable<object[]> JsonEncodedTextStringsCustom
+        {
+            get
+            {
+                return new List<object[]>
+                {
+                    new object[] { "age", "\\u0061\\u0067\\u0065" },
+                    new object[] { "éééééêêêêê", "éééééêêêêê" },
+                    new object[] { "ééééé\"êêêêê", "ééééé\\u0022êêêêê" },
```

Comment:
Why are we unicode-escaping a double-quote?!  JSON says to just escape it.

See [json.org](https://www.json.org/json-en.html) "strings" section.